packages/settings/store/middleware.js

```diff
@@ -9,6 +9,7 @@ const settings = store => next => (action) => {
   switch (action.type) {
     case actionTypes.networkConfigSet:
       // store.dispatch(pricesRetrieved());
+      if(!action.data) break;
```

Comment:
it isn't better to write "break" in the second line so eyes can catch it easily?
```suggestion
       if(!action.data) {
            break;
       }
```